docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+postgres:
+  image: postgres:9.3
+  environment:
+    POSTGRES_PASSWORD: mysecretpassword
+    POSTGRES_USER: postgres
+redis:
+  image: redis
+web:
+  build: .
+  volumes:
+    - ./:/opt/hummingbird/
+  ports:
+    - 3000
```

Comment:
It was to avoid conflicts with other applications.
I changed that.

Do you want 4200 to be fixed too?